2021/1xxx/CVE-2021-1965.json

```diff
@@ -1,18 +1,69 @@
 {
```

Comment:
An exception of type JSONDecodeError occurred. Arguments:

Reply:
Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues